Parsers/ASimDns/Parsers/ASimDnsMicrosoftNXlog.yaml

```diff
@@ -211,11 +268,17 @@ ParserQuery: |
         , DnsQueryType between (261 .. 32767), 'Unassigned'
         , 'Unassigned'),
       EventResult=iff (EventResult == "Based on RCODE", iff(DnsResponseCode == 0, "Success", "Failure"),EventResult)
-   | extend
+    | extend
     // Aliases
       IpAddr = SrcIpAddr,
-      Src = SrcIpAddr
+      Src = SrcIpAddr,
+    // Backward compatibility
+      Query = DnsQuery,
+      QueryType = DnsQueryType,
+      QueryTypeName = DnsQueryTypeName,
+      ResponseCode = DnsResponseCode,
+      ResponseCodeName = DnsResponseCodeName
   | project-away
       *_s, *_d, QTypeName, TenantId, SourceSystem, MG, ManagementGroupName, Computer, RawData
-    };
-    ASimDnsMicrosoftNXLog(disabled)
\ No newline at end of file
+  };
+  ASimDnsMicrosoftNXLog(disabled)
```

Comment:
please update as below:

ASimDnsMicrosoftNXLog(disabled=disabled)

Reply:
Shouldn't that rather be
```ASimDnsMicrosoftNXLog(disabled=false)```

Reply:
no. see this for reference: https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoASA.yaml

Reply:
fixed